wordpress-mobile/WordPress-Android
# Update zendesk version to 5.1

This PR updates zendesk from 2.0 to 5.1. I've checked the change log and it seems like there shouldn't be any breaking changes except renaming of UiConfig to Configuration (https://developer.zendesk.com/embeddables/docs/android-support-sdk/release_notes). 

For some reason `MaterialAlertDialogBuilder` required non-null activity as a parameter after the library bump but it felt like it causes no harm.

To test:
- Go to the Help activity
- Try contacting support
- Try checking your previous tickets
- Try changing email
- Everything should work without change

PR submission checklist:
